scalding-core/src/main/scala/com/twitter/scalding/typed/cascading_backend/CascadingBackend.scala

```diff
@@ -262,12 +266,12 @@ object CascadingBackend {
             val pipe = RichPipe.assignName(pp)
             fd.addTrap(pipe, sink.createTap(Write)(mode))
             CascadingPipe[u](pipe, sink.sinkFields, fd, conv)
-          case (WithDescriptionTypedPipe(input, descr, dedup), rec) =>
+          case (WithDescriptionTypedPipe(input, descr, dedup, quoted), rec) =>
 
             @annotation.tailrec
             def loop[A](t: TypedPipe[A], acc: List[(String, Boolean)]): (TypedPipe[A], List[(String, Boolean)]) =
               t match {
-                case WithDescriptionTypedPipe(i, desc, ded) =>
+                case WithDescriptionTypedPipe(i, desc, ded, quoted) =>
                   loop(i, (desc, ded) :: acc)
```

Comment:
don't we need to accumulate the `quoted`? Seems like we need a `Semigroup` on quoted (maybe a `Monoid`).

Reply:
@johnynek `desc` already has the description information from `quoted` (see `TypedPipe.withQuoted`). I think we don't need to accumulate `quoted` at this point since the purpose is to set the description only (no projections).